Bottom: 3913d172c2d623b9aa46a39a6b725092c2b0d3ca
Top:    43f639f6fd5107bf4b50f4200b055136fa2925b5
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-21 16:47:14 -0700

Refresh of conn-err

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 34ff421..d2f4a9b 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -484,7 +484,8 @@ static int client_connect(void)
 		if (ret)
 			goto err;
 		if (err) {
-			ret = rdma_seterrno(err);
+			ret = -1;
+			errno = err;
 			perror("async rconnect")
 			goto err;
 		}
